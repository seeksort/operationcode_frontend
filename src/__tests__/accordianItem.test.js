import React from 'react';
import renderer from 'react-test-renderer';
import AccordianItem from './../shared/components/accordian/accordianItem/accordianItem';

it('renders correctly', () => {
  const tree = renderer.create(
    <AccordianItem>
      example
    </AccordianItem>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
