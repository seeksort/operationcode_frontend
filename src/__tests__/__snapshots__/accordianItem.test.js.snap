// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<dt
  className={undefined}
>
  <a
    aria-controls="accordion1"
    aria-expanded="false"
    className={undefined}
  >
    First Accordion heading
  </a>
</dt>
`;
